components/skus/browser/rs/lib/src/sdk/credentials/mod.rs

```diff
@@ -100,6 +110,7 @@ where
                             remaining_credential_count: 1,
                             expires_at,
                             active: true,
+                            next_batch_active_at: None,
```

Comment:
I only implemented next_batch_active_at for time limited v2, since that's the only place it's needed.

Reply:
I might suggest calling this `next_active_at` to keep it generic ( even though it is only currently implemented for tlv2 creds. )